client/my-sites/google-my-business/select-location/index.js

```diff
@@ -73,7 +78,7 @@ class GoogleMyBusinessSelectLocation extends Component {
 					<GoogleMyBusinessLocation isCompact key={ location.id } location={ location }>
 						<Button
 							href={ `/google-my-business/stats/${ siteSlug }` }
-							onClick={ this.trackConnectLocationClick }
+							onClick={ this.connectLocation.bind( null, location.id ) }
```

Comment:
This is generating a linting error, but I couldn't find an elegant way to address it:

```javascript
81:8   error  JSX props should not use .bind()                                                      react/jsx-no-bind
81:18  error  JSX props should not use .bind()                                                      react/jsx-no-bind
```